include/asterisk/chanvars.h

```diff
@@ -33,27 +33,158 @@ struct ast_var_t {
 
 AST_LIST_HEAD_NOLOCK(varshead, ast_var_t);
 
+
+/*!
+ * \brief Create a new variables list and initalize to empty
+ *
+ * \retval NULL on error
+ * \retval returns the pointer to the newly created list on success
```

Comment:
"returns the" is not needed
Do a find and replace on the rest of the file too for "\retval returns "